recipes/perl-retroseq/meta.yaml

```diff
@@ -11,22 +11,28 @@ source:
 
 build:
   noarch: generic
-  number: 1
+  number: 2
   run_exports: {{ pin_subpackage("perl-retroseq", max_pin="x") }}
 
 requirements:
-  build:
+  host:
     - perl
     - perl-carp
     - perl-exporter
     - perl-extutils-makemaker
+    - zlib
+    - ncurses
+    - samtools ==0.1.19
 
   run:
     - perl
     - perl-carp
-    - samtools =0.1.16
-    - exonerate
+    - samtools ==0.1.19
```

Comment:
Does it need to be exactly `0.1.19` or a newer version will work too ?

Reply:
Has to be exact. retroseq is abandonware unfortunately.